TSE/CCExtensions.cpp

```diff
@@ -2439,55 +2473,63 @@ static PRIMITIVEPROCDEF g_Extensions[] =
 			"iv",	PPFLAG_SIDEEFFECTS,	},
 
 		{	"sysFindObject",				fnSystemFind,	FN_SYS_FIND,	
-			"(sysFindObject source filter) -> list of objects\n\n"
+			"(sysFindObject source criteria) -> list of objects\n\n"
 			
-			"criteria\n\n"
+			"criteria is a string that must specify one or more of the following categories:\n\n"
 			
 			"   b           Include beams\n"
 			"   k           Include markers\n"
 			"   m           Include missiles\n"
 			"   s           Include ships\n"
 			"   t           Include stations (including planets)\n"
-			"   z           Include the player\n"
-			"\n"
+			"   T           Include structure-scale stations\n"
+			"   z           Include the player\n\n"
+			
+			"OR one of the following:\n\n"
+			
+			"   G           Include only stargates\n"
+			"   G:xyz       Include only stargate with ID 'xyz'\n"
+			"   t:xyz       Include only stations with attribute 'xyz'\n"
+			"   T:xyz       Include only structure-scale stations with attribute 'xyz'\n\n"
```

Comment:
I'm trying to explain the rules in the simplest possible way that doesn't exclude useful combinations but does exclude combinations that don't produce the expected results (like G with other include options). But apparently "T/t:xyz" is the same as "T/t +xyz", but this doesn't work for other letters. It is used in several places in the core data, so it should be documented. Ick.